editor/editor_tests/xml_feature_test.cpp

```diff
@@ -480,3 +480,89 @@ UNIT_TEST(XMLFeature_Diet)
   ft.SetCuisine("");
   TEST_EQUAL(ft.GetCuisine(), "", ());
 }
+
+UNIT_TEST(XMLFeature_SocialContactsProcessing)
+{
+  {
+    std::string const nightclubStr = R"(<?xml version="1.0"?>
+    <node lat="50.4082862" lon="30.5130017" timestamp="2022-02-24T05:07:00Z">
+    <tag k="amenity" v="nightclub" />
+    <tag k="name" v="Stereo Plaza" />
+    <tag k="contact:facebook" v="http://www.facebook.com/pages/Stereo-Plaza/118100041593935" />
+    <tag k="contact:instagram" v="https://www.instagram.com/p/CSy87IhMhfm/" />
+    <tag k="contact:line" v="liff.line.me/1645278921-kWRPP32q/?accountId=673watcr" />
+    </node>
+    )";
+
+    editor::XMLFeature xmlFeature(nightclubStr);
+
+    osm::EditableMapObject emo;
+    editor::FromXML(xmlFeature, emo);
+
+    // Read and write "contact:facebook" to apply normalization.
+    std::string contactFacebook = std::string(emo.GetMetadata(feature::Metadata::FMD_CONTACT_FACEBOOK));
```

Comment:
Here is the shorter way to construct string from view:
std::string contactFacebook(view);